package.json

```diff
@@ -94,6 +94,27 @@
           "type": "string",
           "default": "# ${noteName}\n\n",
           "description": "Template for auto-created note files. Available tokens: ${noteName}, ${timestamp}. Timestamp is inserted in ISO format, i.e. 2020-07-09T05:29:00.541Z."
+        },
+        "vscodeMarkdownNotes.allowPipedWikiLinks": {
+          "type": "boolean",
+          "default": false,
+          "description": "States whether or not to use piped wikilinks. When set to false, this will ignore any '|' character and slugify it as normal."
+        },
+        "vscodeMarkdownNotes.pipedWikiLinksSyntax": {
+          "type": "string",
+          "enum": [
+            "desc|file",
+            "file|desc"
```

Comment:
Hey, I am guessing you added this setting in response to the question:

> Do Obsidian, Roam Research, Foam, and other projects all treat the characters before the pipe as the file and the characters after as the link text (or does anyone do the inverse)? Seems like your PR follows the mediawiki style and I just want to make sure this how it works for other projects.

I am curious what you found here -- were there some programs that inverted the order here? Did you find any consensus on this?

If there are different ways it works out there, I think the setting is the way to handle.

If all the related programs handle consistently, maybe the setting is just confusing.

Any thoughts on this before we merge?

Reply:
> I am curious what you found here -- were there some programs that inverted the order here? Did you find any consensus on this?

Two things:  [MediaWiki's meta page](https://meta.wikimedia.org/wiki/Help:Piped_link) and [What Github Wiki uses](https://github.com/gollum/gollum/wiki#link-tag)

> If there are different ways it works out there, I think the setting is the way to handle.

Yes, agreed.